demo/src/components/AppSidebar.svelte

```diff
@@ -16,7 +16,7 @@
       </header>
     {/if}
 
-    <article class="p-5 lg:p-10 flex flex-col gap-1.5 lg:gap-3" class:pt-0={!noHeader}>
+    <article class="p-5 lg:p-10 flex flex-col gap-1.5 lg:gap-3" class:!pt-0={!noHeader}>
```

Comment:
This fixes the padding on large screens using the `!important`. Not the most elegant solution, but it'll do.

Reply:
OK